fuse/direntry.go

```diff
@@ -30,6 +30,24 @@ func (d DirEntry) String() string {
 	return fmt.Sprintf("%o: %q", d.Mode, d.Name)
 }
 
+// DirEntryIno is a DirEntry struct extended by the "Ino" field for the
+// inode number.
+type DirEntryIno struct {
```

Comment:
why? can't you simply add the Ino field to DirEntry? We could translate the default value 0 to FUSE_UNKNOWN_INODE.